app/views/cases/offender_sar/_requester_details_step.html.slim

```diff
@@ -8,7 +8,9 @@
 
 = form_for @case, url: case_create_action(@case.object), as: :offender_sar do |f|
   #third-party
-    = f.radio_button_fieldset(:third_party) do |fieldset|
+    h2.heading-medium
+      = t('helpers.label.offender_sar_case_form.third_party')
+    = f.radio_button_fieldset(:third_party, legend_options: { class: 'visually-hidden' }) do |fieldset|
       - fieldset.radio_input(true, text_method: :humanize)
```

Comment:
Note that this follows with the field below about delivery method. For accessibility the label was changed to make sense in en.local. An alternative would be to change the label and update the styles, removing the h2, but the heading would then be lost, making it inconsistent with below section.